components/application-deployers/org.wso2.carbon.application.deployer.synapse/src/main/java/org/wso2/carbon/application/deployer/synapse/SynapseAppDeployerConstants.java

```diff
@@ -32,6 +32,7 @@
     public static final String API_TYPE = "synapse/api";
     public static final String TEMPLATE_TYPE = "synapse/template";
     public static final String INBOUND_ENDPOINT_TYPE = "synapse/inbound-endpoint";
+    public static final String SYNAPSE_LIBRARY_TYPE = "zip";
```

Comment:
At this moment we can't test this with other artifact type (rather than "zip"). I think synapse/lib would be better. But It should be supported by developer studio. After releasing the feature for this, we can change this. If I use "synapse/lib" while creating the CApp, the CApp will not contain connector zip.